content/references/http-websocket-apis/public-api-methods/server-info-methods/server_info-clio.md

```diff
@@ -58,63 +51,202 @@ An example of a successful response:
+                        "errored": "11",
+                        "forwarded": "0",
+                        "duration_us": "6179145"
+                    },
+                    "ripple_path_find": {
+                        "started": "1",
+                        "finished": "1",
+                        "errored": "0",
+                        "forwarded": "1",
+                        "duration_us": "1409563"
+                    },
+                    "account_channels": {
+                        "started": "14",
+                        "finished": "14",
+                        "errored": "0",
+                        "forwarded": "0",
+                        "duration_us": "1062692"
+                    },
+                    "submit": {
+                        "started": "6",
+                        "finished": "6",
+                        "errored": "0",
+                        "forwarded": "6",
+                        "duration_us": "11383"
+                    },
+                    "transaction_entry": {
+                        "started": "8",
+                        "finished": "5",
+                        "errored": "3",
+                        "forwarded": "0",
+                        "duration_us": "494131"
+                    }
+                },
+                "subscriptions": {
+                    "ledger": 0,
+                    "transactions": 0,
+                    "transactions_proposed": 0,
+                    "manifests": 2,
+                    "validations": 2,
+                    "account": 0,
+                    "accounts_proposed": 0,
+                    "books": 0
+                }
+            },
+            "load_factor": 1,
+            "clio_version": "0.3.0-b2",
+            "validation_quorum": 8,
+            "rippled_version": "1.9.1-rc1",
+            "validated_ledger": {
+                "age": 4,
+                "hash": "4CD25FB70D45646EE5822E76E58B66D39D5AE6BA0F70491FA803DA0DA218F434",
+                "seq": 19977628,
+                "base_fee_xrp": 1E-5,
+                "reserve_base_xrp": 1E1,
+                "reserve_inc_xrp": 2E0
+            }
         },
-        "disconnected": {
-          "duration_us": "1207534",
-          "transitions": 2
+        "cache": {
+            "size": 8812733,
+            "is_full": true,
+            "latest_ledger_seq": 19977629
         },
-        "full": {
-          "duration_us": "3589270527034",
-          "transitions": 2
+        "etl": {
+            "etl_sources": [
+                {
+                    "validated_range": "19405538-19977629",
+                    "is_connected": "1",
+                    "ip": "52.36.182.38",
+                    "ws_port": "6005",
+                    "grpc_port": "50051",
+                    "last_msg_age_seconds": "0"
+                }
+            ],
+            "is_writer": true,
+            "read_only": false,
+            "last_publish_age_seconds": "2"
```

Comment:
I think both of these are likely to be moved to admin only sometime soon, and we should probably not document them